src/NuGet.Core/NuGet.Configuration/Proxy/ProxyCache.cs

```diff
@@ -99,6 +99,14 @@ public WebProxy GetUserConfiguredProxy()
                 {
                     webProxy.Credentials = new NetworkCredential(userName, password);
                 }
+                
+                var noProxy = _settings.GetValue(SettingsUtility.ConfigSection, ConfigurationContants.NoProxy);
```

Comment:
I tested locally by running build.ps1 and examined the build log.

Unfortunately, I got some "false positive" failures (https://github.com/NuGet/Home/issues/1998) which made it difficult to understand if the failures were "mine" or not.

comparing the log against a "dev" build.